packages/services/f-braze-adapter/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@justeat/f-braze-adapter",
   "description": "Fozzie Metadata Service",
-  "version": "4.0.0-beta.2",
-  "main": "src/index.js",
+  "version": "4.0.0-beta.3",
+  "main": "dist/f-braze-adapter.umd.js",
```

Comment:
You could also add a `"module": "dist/f-braze-adapter.es.js"` line to offer ESM to package consumers.

Reply:
good idea have added